src/main/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -574,7 +634,10 @@ public VariantContext make() {
     }
 
     public VariantContext make(final boolean leaveModifyableAsIs) {
-        if(!leaveModifyableAsIs) attributesCanBeModified = false;
+        if(!leaveModifyableAsIs){
```

Comment:
Add whitespace please
```suggestion
        if (!leaveModifyableAsIs) {
```